parsers/PH.py

```diff
@@ -617,9 +617,13 @@ def pivot_per_mode(df: pd.DataFrame) -> pd.DataFrame:
     # Flatten columns and make them "production.{mode}"
     df.columns = df.columns.to_series().str.join(".")
     # Handle storage
-    if "production.hydro_storage" in df.columns:
-        df = df.rename(columns={"production.hydro_storage": "storage.hydro"})
-        df["storage.hydro"] *= -1
+    storage_methods = {"hydro_storage": "hydro", "battery": "battery"}
+    for storage_method, storage_mode in storage_methods.items():
+        if f"production.{storage_method}" in df.columns:
+            df = df.rename(
+                columns={f"production.{storage_method}": f"storage.{storage_mode}"}
+            )
+            df[f"storage.{storage_mode}"] *= -1
```

Comment:
Don't this run the risk of attributing all hydro to hydro storage?

I would prefer if we split up the initial plant mapping into production and storage instead so we can assign them on a per plant basis. This would also facilitate fetching the negative values that we talked about. (and avoid string matching that is very prone to mistakes)

Reply:
> Don't this run the risk of attributing all hydro to hydro storage?

I don't see how, here we would have `production.hydro` stay `production.hydro` and `production.hydro_storage` become `storage.hydro`

> I would prefer if we split up the initial plant mapping into production and storage

That could be best indeed but the scope of the change is bigger

Reply:
Let's merge this then and get the fix implemented and then we can look into changing the mappings later on.